### PR TITLE
Add the project's maintainers info

### PR DESCRIPTION
**Pull request by olivierobert**
## What happened 👀

Add `CODEOWNERS` in the subdirectory `.github`

## Insight 📝

All project template repositories now have official leads so it is crucial to have it documented. In addition, this will ensure that all concerned developers are notified when new pull requested are created ✌️ 

## Proof Of Work 📹

I needed to add the team to the repository:

image

There are no issues with the file format since it is used on the other project repositories.

### CODE REVIEW

**Review by hoangmirs — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Team Lead
+* @junan
+
+# Team Members
+* @carryall @hoangmirs @Lahphim @malparty
```
Comment (hoangmirs) [inline .github/CODEOWNERS]: @olivierobert 
I've just found that this line will override the above line, so Junan won't be the code owner of this project.
This is the preview when I open a PR on react-templates:
__53__Add_react-router_to_the_template_by_hoangmirs_·_Pull_Request__61_·_nimblehq_react-templates_🔊
And it did not include Pae.

https://github.community/t/order-of-owners-in-codeowners-file/143073

How about adding the team lead at the beginning of each list instead?

**Review by hoangmirs — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (malparty) [reply .github/CODEOWNERS]: Nice catch!! 💡

Comment (malparty) [reply .github/CODEOWNERS]: I agree with Hoand recommendation. To keep it readable, we might want to comment this single line with:
`#  Team Lead / Team Members`

Comment (olivierobert) [reply .github/CODEOWNERS]: Oh, I thought they would be compounded rules 😭  So yes, we would need to use some kind of comments as suggested by Xavier but I would prefer to be more detailed:

```
# Team
# @junan is the Team Lead and the others are team members
* @junan @carryall @hoangmirs @Lahphim @malparty

# Engineering Leads
CODEOWNERS @nimblehq/engineering-leads
```

Comment (hoangmirs) [reply .github/CODEOWNERS]: @olivierobert Do we need to add the team lead to review `CODEOWNERS` file? As he/she might be the one who merges the PR 🤔 
```
# Team
# @junan is the Team Lead and the others are team members
* @junan @carryall @hoangmirs @Lahphim @malparty

# Engineering Leads
CODEOWNERS @nimblehq/engineering-leads @junan
```

Comment (olivierobert) [reply .github/CODEOWNERS]: Actually, no need, as the previous rules will ensure that the whole team is added as reviewers.

Comment (luongvo) [reply .github/CODEOWNERS]: @olivierobert @hoangmirs Confirmed that we don't need to add TL at `CODEOWNERS` pattern. Time to make updates on all repositories again. 😢 

image

Comment (olivierobert) [reply .github/CODEOWNERS]: Indeed @luongvo, all template repositories need to be updated again 😭 The issue was not found on [the initial PR](https://github.com/nimblehq/rails-templates/pull/328) to add the CODEOWNER file

Comment (hoangmirs) [reply .github/CODEOWNERS]: I modified it following the suggestion in https://github.com/nimblehq/gin-templates/pull/75/commits/f1d03851d1208f6dfa2934de291cf67c17c51880

**Review by hoangmirs — APPROVED** [review]

**Review by Lahphim — APPROVED** [review]

**Pull request merged**